toqito/helper/update_odometer.py

```diff
@@ -32,8 +32,8 @@ def update_odometer(
     >>> vec = np.array([0, 0])
     >>> upper_lim = np.array([3, 2])
     >>> for j in range(0, np.prod(upper_lim)-1):
-    >>> vec = update_odometer(vec, upper_lim)
-    >>> vec
+    ...     vec = update_odometer(vec, upper_lim)
```

Comment:
Hmm, maybe I don't quite understand, but is there a reason it went from `>>>` to `...`?

Reply:
`>>>` is a new line according to doctest whereas `...` is considered a continued line. 

In this case, you start with a `for` loop. I used `...` for the next line because that's what the for loop is for.